src/app/store.ts

```diff
@@ -1,10 +1,20 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
```

Comment:
Don't disable eslint. Using cycle import is not a good practice. Try to solve this error, please